resources/helm/v2.6/istio-control/istio-discovery/values.yaml

```diff
@@ -598,6 +598,13 @@ istio_cni:
   enabled: false
   chained: true
 
+# Gateway Settings
+gateways:
+  # Define the security context for the pod.
+  # If unset, this will be automatically set to the minimum privileges required to bind to port 80 and 443.
+  # On Kubernetes 1.22+, this only requires the `net.ipv4.ip_unprivileged_port_start` sysctl.
+  securityContext: {}
+
```

Comment:
I don't understand why the changes here or to files in `resources/helm/v2.6/istio-control/istio-discovery/` are included in this PR. Were these all generated because of a changes in a previous PR?

Reply:
Yes. These were pulled by `make gen`. They will disappear now that https://github.com/maistra/istio-operator/pull/1820 has been merged.

Reply:
Thanks!